testsuite/shared/src/main/resources/application-users.properties

```diff
@@ -6,7 +6,7 @@
 # guest=guest
 # user1=password1
 # user2=password2
-#remoteejbuser=rem@teejbpasswd1
+# remoteejbuser=rem@teejbpasswd1
```

Comment:
Is there a reason for this additional space?  I don't see where this line gets uncommented.

Reply:
The reason legibility and to follow the same convention as the example `# login=password` above.

It doesn't get uncommented since the password is already encoded on line 13. This is un-encoded version of that password.

Reply:
Thank you, GitHub cut off at line 12 - I see now.